lib/project_types/script/commands/create.rb

```diff
@@ -5,6 +5,12 @@ class Command
     class Create < ShopifyCLI::Command::SubCommand
       prerequisite_task :ensure_authenticated
 
+      recommend_node(
+        from: ::Script::Layers::Infrastructure::Languages::TypeScriptProjectCreator::MIN_NODE_VERSION,
+        to: ShopifyCLI::Constants::SupportedVersions::Node::TO
+      )
```

Comment:
Similar to my comment above, I believe we only require node if the user's script project requires it (e.g. a TypeScript project). I think this comment also applies to `push` below. cc @jacobsteves @lopert